pyvista/plotting/renderer.py

```diff
@@ -3357,6 +3358,9 @@ def add_legend(
             or ``'arial'``. Defaults to :attr:`pyvista.global_theme.font.family
             <pyvista.plotting.themes._Font.family>`.
 
+        background_opacity : int | float, default: 1.0
```

Comment:
I'm assuming that it should be a float input, but an int can be promoted to a float in python/vtk?

```suggestion
        background_opacity : float, default: 1.0
```

Reply:
Good catch. Thanks.